test/views/ViewArrivalsSpec.scala

```diff
@@ -75,9 +77,48 @@ class ViewArrivalsSpec extends SingleViewSpec("viewArrivals.njk") with Generator
     ls.eq(5).text() mustBe "11 August 2020"
   }
 
+  val rows: Elements = doc.getElementsByAttributeValue("role", "row")
+
   "generate a row for each arrival" in {
-    val ls: Elements = doc.getElementsByAttributeValue("role", "row")
+    rows.size() mustEqual 7
+  }
 
-    ls.size() mustEqual 7
+  "display rows in correct (sorted) order" in {
+    rows.toList.zipWithIndex.forEach {
+      x =>
+        val cells: Elements = x._1.getElementsByAttributeValue("role", "cell")
+
+        cells.get(0).text() mustBe "viewArrivalNotifications.table.updated " + (x._2 match {
+          case 0 => "6:06am"
+          case 1 => "5:05am"
+          case 2 => "4:04am"
+          case 3 => "3:03am"
+          case 4 => "2:02am"
+          case 5 => "1:01am"
+          case 6 => "1:00am"
+        })
+    }
```

Comment:
I like that we're asserting on the elements themselves here but I don't think the test is clear. How about using a [Scalatest matcher](https://www.scalatest.org/user_guide/using_matchers#workingWithAggregations) like `theSameElementsAs`?